app-accessibility/emacspeak-ss/emacspeak-ss-1.9.1.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
```

Comment:
Please revbump for new EAPIs and drop to ~arch.

There's a few reasons:
1. Clear out stale stuff in the vdb. One day, we'd like to be able to drop support for older EAPIs from Portage. This helps do that.
2. Newer EAPIs change behaviour, e.g. passing additional args into configure by default, and we want to find any problems from that.
3. When porting to a new EAPI, you should take the opportunity to do a general tidy (and check for any open easy bugs to fix for the package).